source/jobs/data-verification/data-verification.txt

```diff
@@ -2,6 +2,38 @@
 Data Verification
 =================
 
+Data verification is a process which is built into the sync engine.
```

Comment:
([that vs. which](https://www.mongodb.com/docs/meta/style-guide/writing/use-that-which-correctly/)) 
Nit. Should have marked this the first time. S/b: "... is a process that is ..."